openfe/protocols/openmm_septop/equil_septop_method.py

```diff
@@ -0,0 +1,2125 @@
+
+        # Ensure that number of overall lambda windows matches number of lambda
+        # windows for individual components
+        if n_replicas != len(lambda_vdw_B):
+            errmsg = (f"Number of replicas {n_replicas} does not equal the"
+                      f" number of lambda windows {len(lambda_vdw_B)}")
+            raise ValueError(errmsg)
+
+        # Check if there are lambda windows with naked charges
+        for inx, lam in enumerate(lambda_elec_A):
+            if lam < 1 and lambda_vdw_A[inx] == 1:
+                errmsg = (
+                    "There are states along this lambda schedule "
+                    "where there are atoms with charges but no LJ "
+                    f"interactions: Ligand A: lambda {inx}: "
+                    f"elec {lam} vdW {lambda_vdw_A[inx]}")
+                raise ValueError(errmsg)
+            if lambda_elec_B[inx] < 1 and lambda_vdw_B[inx] == 1:
+                errmsg = (
+                    "There are states along this lambda schedule "
+                    "where there are atoms with charges but no LJ interactions"
+                    f": Ligand B: lambda {inx}: elec {lambda_elec_B[inx]}"
+                    f" vdW {lambda_vdw_B[inx]}")
+                raise ValueError(errmsg)
+
+    def _create(
+        self,
+        stateA: ChemicalSystem,
+        stateB: ChemicalSystem,
+        mapping: Optional[Union[gufe.ComponentMapping, list[gufe.ComponentMapping]]] = None,
+        extends: Optional[gufe.ProtocolDAGResult] = None,
+    ) -> list[gufe.ProtocolUnit]:
+        # TODO: extensions
+        if extends:  # pragma: no-cover
+            raise NotImplementedError("Can't extend simulations yet")
+
+        # Validate components and get alchemical components
+        self._validate_complex_endstates(stateA, stateB)
+        alchem_comps = system_validation.get_alchemical_components(
+            stateA, stateB,
+        )
+        self._validate_alchemical_components(alchem_comps)
+
+        # Validate the lambda schedule
+        self._validate_lambda_schedule(self.settings.solvent_lambda_settings,
+                                       self.settings.solvent_simulation_settings)
+        self._validate_lambda_schedule(self.settings.complex_lambda_settings,
+                                       self.settings.complex_simulation_settings)
+
+        # Check nonbonded and solvent compatibility
+        nonbonded_method = self.settings.forcefield_settings.nonbonded_method
+        # Use the more complete system validation solvent checks
+        system_validation.validate_solvent(stateA, nonbonded_method)
+
+        # Validate solvation settings
+        settings_validation.validate_openmm_solvation_settings(
+            self.settings.solvent_solvation_settings
+        )
+
+        # Make sure that we have the full system for restraint trajectory analysis
+        if self.settings.complex_equil_output_settings.output_indices != "all":
```

Comment:
If this is hard set to be all, we probably should not expose it as an option or ignore what the user says, I think that would lead to a better user experience!

Reply:
This does need rethinking, but given it would require changing a ton of settings, I would suggest punting this to a separate issue.

Reply:
We ideally need something that will work for both ABFEs and SepTop. Right now I don't have good insight into what that is. I would say this is something we can do as a separate issue that we can do before we release v1.6.